includes/definitions/stulz.yaml

```diff
@@ -4,6 +4,46 @@ type: environment
 icon: stulz
 over:
     - { graph: device_bits, text: 'Device Traffic' }
+snmp_bulk: true
 discovery:
     - sysObjectID:
-        - .1.3.6.1.4.1.29462.
+        - .1.3.6.1.4.1.29462.10
```

Comment:
The oid you removed matches the one you added and yo likely removed support for other devices.